ludwig/data/dataset/pandas.py

```diff
@@ -74,6 +74,12 @@ def get_dataset(self):
     def __len__(self):
         return self.size
 
+    @property
+    def in_memory_size_bytes(self):
+        df = self.to_df()
+        memory_usage = df.memory_usage(deep=True).sum() if not None else 0
```

Comment:
`if not None` will always evaluate to `True`

Reply:
Good catch, will change this